svthalia/concrexit
# Use randomized filenames for all photos

Closes #3442.

### Summary
After this, filenames should no longer sometimes be ugly.

### How to test

1. Run the reuploading script I posted in #3324.
2. Check that the files are reuploaded with nice random hex names.
3. Upload some more files, and check their filenames.
